app/assets/stylesheets/_navbar.scss

```diff
@@ -1,66 +1,107 @@
 .navbar {
+  padding: 0 10px;
   margin-bottom: 0;
-  min-height: 0;
-  font-size: 12px;
-  background: $lightgrey-2;
-  border: 0;
+  color: $white;
   border-radius: 0;
 
-  .nav-pills {
-    .nav-item {
-      &.active {
-        .nav-link {
-          background-color: transparent;
-          border-bottom: 2px solid $darkred;
-          color: $darkred;
-        }
-      }
+  &-logo {
+    padding: 10px 15px;
+  }
+
+  &-nav > li > a {
+    padding: 15px 10px 15px 5px;
+    font-weight: 600;
+  }
+
+  &-header {
+    width: 100%;
+  }
+
+  &-btn {
+    border-color: transparent;
+
+    &-reports {
+      margin-left: 15px;
+    }
+  }
+
+  &-right {
+    margin-right: 0;
+  }
+
+  &-right &-locale,
+  &-right .dropdown {
+    display: inline-block;
+  }
+
+  &-form {
+    padding: 0;
+  }
 
-      .nav-link {
-        border-radius: 0;
-        padding: 5px 15px;
-        min-width: 80px;
-        text-align: center;
-        text-transform: uppercase;
-        font-weight: 800;
-        color: $darkgrey-2;
-        transition: 0.4s ease all;
-
-        &:focus {
-          background-color: inherit;
-        }
-
-        &:before {
-          content: ' ';
-          width: 1px;
-          position: absolute;
-          height: 55%;
-          background-color: $lightgrey-3;
-          right: 0;
-        }
-
-        &:after {
-          background: $darkred;
-          bottom: -2px;
-          content: ' ';
-          height: 2px;
-          left: 0;
-          position: absolute;
-          transform: scaleX(0);
-          transform-origin: 50% 50%;
-          transition: 0.3s ease all;
-          width: 100%;
-        }
-
-        &:hover {
-          background: transparent;
-          color: $darkred;
-
-          &:after {
-            transform: scaleX(1);
-          }
-        }
+  &-search {
+    margin: 10px;
+  }
+
+  .velocity {
+    position: relative;
+    padding-left: 15px;
+    margin-left: 5px;
+
+    &:after {
+      content: '';
+      position: absolute;
+      top: 1px;
+      left: 0;
+      width: 1px;
+      height: 20px;
+      background-color: rgba($darkgrey-6, .25);
+    }
+  }
+
+  &-locale {
+    margin: 15px 10px;
+    font-size: 14px;
+    color: $darkgrey-6;
+
+    .locale-change {
+      position: relative;
+      padding: 0 20px 0 0;
+      background-color: transparent;
+      border: none;
+      border-radius: 0;
+      appearance: none;
+    }
+
+    .locale-change-wrapper {
+      position: relative;
+
+      &:after {
+        content: '';
+        position: absolute;
+        top: 9px;
+        right: 5px;
+        display: inline-block;
+        width: 0;
+        height: 0;
+        vertical-align: middle;
+        border-top: 4px dashed;
+        border-top: 4px solid \9;
```

Comment:
It is! Check it out:
![screen shot 2017-01-19 at 2 07 14 pm](https://cloud.githubusercontent.com/assets/463661/22116861/99c4a90e-de50-11e6-9e27-16cf05674526.png)